charts/deployer/templates/deployment.yaml

```
@@ -11,6 +11,10 @@ spec:
   selector:
     matchLabels:
       {{- include "helpers.selectorLabels" . | nindent 6 }}
+  {{- with .Values.strategy }}
+  strategy:
+    {{- toYaml . | nindent 8 }}
```

Indentation should be 4 I think
```suggestion
    {{- toYaml . | nindent 4 }}
```